StripeConnect/StripeConnect/Source/Internal/Webview/MessageSenders/CallSetterWithSerializableValueSender.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  CallSetterWithSerializableValueSender.swift
+//  StripeConnect
+//
+//  Created by Chris Mays on 8/7/24.
+//
+
+import Foundation
+
+struct CallSetterWithSerializableValueSender: MessageSender {
+    struct Message: Codable, Equatable {
+        let setter: String
+        let value: String
```

Comment:
optional naming nit + docstrings:
```suggestion
    struct Payload: Encodable, Equatable {
        /// Name of the component-specific JS setter function on the component (e.g. `setFullTermsOfService`)
        let setter: String
        /// Args value to pass to the setter function
        let value: Value
```